### ISSUE TITLE
Why the return format of the output differs when the args.strategy varies？

### ISSUE THREAD

**Issue by keqixiaojie** [thu-coai/Emotional-Support-Conversation#40]
你好！我在看BlenderEmotionSupport.py中的train函数时，发现args.strategy不同，output返回格式不一样
`if not args.strategy:   #非策略返回序列
    outputs = model(input_ids, attention_mask = input_ids.ne(tokenizer.pad_token_id), 
								    decoder_input_ids=decoder_input_ids, decoder_turn_ids=decoder_turn_ids, 
								    decoder_role_ids=decoder_role_ids, turn_ids=turn_ids, role_ids=role_ids,
								    labels = decoder_label_ids)
    ppl = loss = outputs[0]  # model outputs are always tuple in transformers (see doc)                
else:   #策略返回Seq2SeqLMOutput对象（默认）
    outputs = model(input_ids, attention_mask = ..., labels = decoder_label_ids)
    loss = ppl = outputs.loss`
于是我又看了BlenderbotSmallForConditionalGeneration的forward函数实现，发现返回格式只和return_dict参数有关，但是我没有找到args.strategy和return_dict的关联，请问是怎么回事？

还有一个和返回格式相关的问题是：在args.strategy==True时返回的是Seq2SeqLMOutput对象，所以这行代码代表什么？求指导！
`if  args.strategy:
                tr_loss1 += outputs[1].mean().item()`
